examples/How_to_call_functions_with_chat_models.ipynb

```diff
@@ -240,7 +239,7 @@
     "chat_response = chat_completion_request(\n",
     "    messages, tools=tools\n",
     ")\n",
-    "assistant_message = chat_response.json()[\"choices\"][0][\"message\"]\n",
+    "assistant_message = json.loads(chat_response.model_dump_json())[\"choices\"][0][\"message\"]\n",
```

Comment:
No need to convert to JSON and reparse. You can simply access the object directly with 
`chat_response.choices[0].message`

Reply:
I made a change to call chat_response.choices[0].message directly, but for printing the results, I think it still looks cleaner to convert to json.

Also in the latter part of the cookbook when executing SQL queries, it still seemed necessary to convert to json because some pieces of the message were json whereas others were ChatCompletion types